analyzer/runtime/runtime.go

```diff
@@ -283,16 +276,11 @@ func (m *Main) processRound(ctx context.Context, round uint64) error {
 }
 
 // prepareBlockData adds block data queries to the batch.
-func (m *Main) prepareBlockData(ctx context.Context, round uint64, batch *storage.QueryBatch) error {
-	data, err := m.cfg.Source.BlockData(ctx, round)
-	if err != nil {
-		return err
-	}
-
-	for _, f := range []func(context.Context, *storage.QueryBatch, *storage.RuntimeBlockData) error{
+func (m *Main) prepareBlockData(batch *storage.QueryBatch, data *storage.RuntimeAllData) error {
+	for _, f := range []func(*storage.QueryBatch, *storage.RuntimeBlockData) error{
```

Comment:
Nit: WDYT about inlining here? I think the whole func is just `return queueBlockAndTransactionInserts(batch, data.blockData)`.

Reply:
Good point, updated!